packages/formatter-summary/README.md

```diff
@@ -0,0 +1,14 @@
+# Formatter summary (`@sonarwhal/formatter-summary`)
+
+The `summary` formatter prints the results of a site analysis in a table with
+how many errors or warnings a rule has found:
+
+![Example output for the summary formatter](images/summary-output.png)
```

Comment:
Move `images/` into `src/` so that they are copied into `dist/` and get publish with the rest.

Reply:
Done, I've changed the `package.json`